migrations/20230816085700_collections_and_more.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE collections (
+    id bigint PRIMARY KEY,
+    title varchar(255) NOT NULL,
+    description varchar(2048) NOT NULL,
+    team_id bigint REFERENCES teams NOT NULL,
+    published timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    slug varchar(255) NULL UNIQUE,
+    body varchar(65536) NOT NULL DEFAULT ''::varchar,
+
+    public boolean NOT NULL DEFAULT false,
```

Comment:
for this I think it's better to use an enum so this should be a varchar. that way we can have dynamic collection statuses